_posts/2024-05-13-april-rubygems-updates.md

```diff
@@ -0,0 +1,80 @@
+    
+- [Resolved Musl Platform Issues for RubyGems and Bundler](https://github.com/rubygems/rubygems/pull/7583): Since introducing support for the musl platform, there's been different issues and regressions with it, leading to hesitancy among gem authors about releasing musl variants. The maintainer of Nokogiri has been actively identifying these issues, including a critical problem he believes to be the last barrier to fully supporting musl precompiled gems. Addressing this issue seemed necessary. The non-transitivity of `Gem::Platform#===` with musl was causing missing platforms in the lockfile, leading to resolution errors. The issue has been resolved by specifically accommodating the unique aspects of musl when removing invalid platforms from the lockfile.
+
+In April, RubyGems gained [122 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2024-4-01%7D...master@%7B2024-4-31%7D) contributed by 13 authors. There were 1,307 additions and 878 deletions across 111 files.
+
+
+## RubyGems.org News
+
+The updates made this month to RubyGems.org reflect a strong commitment to improving user experience, enhancing security, and modernizing the platform.
+
+The following are highlights of what the team worked on this month:
+
+- [RubyGems Jank Limits](https://github.com/rubygems/rubygems.org/pull/4631):  To reduce the likelihood of disruption caused by a left-pad-like package removal, we've introduced limits on deleting old or highly downloaded gems. Gem deletions are now primarily for immediate fixes of newly released but broken gems, where reverting is the best solution. For other issues, the recommended approach is to release a new version. We've set a provisional limit on gems that can be yanked without a public review to avoid premature exposure. This policy affects gems with over 100,000 downloads or those older than 30 days, aligning more closely with other ecosystems that restrict deletions. We will adjust the policy based on feedback and continue to coordinate yank requests through RubyGems staff, balancing the needs of maintainers and the wider community.
+
+- [Re-introducting Avatars to RubyGems.org](https://github.com/rubygems/rubygems.org/pull/4599):  Originally, profile images were removed from RubyGems.org due to privacy concerns, as Gravatar's system exposing user emails, leading to complaints. This change, however, made the site appear anonymous, diminishing the perceived trustworthiness of gem info pages. To address this issue, [@segiddins](https://github.com/segiddins) has developed a solution that allows images to be safely displayed without compromising privacy. This new method proxies images through RubyGems.org, maintaining user privacy while enhancing the visual appeal and trust of the platform.
```

Comment:
Can you raise this one to the top of the rubygems.org list.

"Gravatar's system _exposed_ user emails"